.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "cpp/discord-rpc"]
 	path = cpp/discord-rpc
-	url = https://github.com/discord/discord-rpc
+	url = https://github.com/cborac/discord-rpc
```

Comment:
Is this a requirement for the buttons to work?

Reply:
Yes, the current repo handled by discord is deprecated and the presence buttons were introduced after the deprecation. Thus, discord's repo does not have an implementation for the buttons

Reply:
Thanks for making that fork - it's a lot easier than rewriting this to work with the new Game SDK. I'd prefer to use an official library but since that's not possible, I'm fine with this so long as it's linked to a specific commit.